src/components/UI/availability_zones_label.js

```diff
@@ -46,14 +46,13 @@ const Wrapper = styled.abbr`
   }
 `;
 
-const AvailabilityZonesLabel = ({ key, label, letter, title }) => (
-  <Wrapper className={letter} key={key} title={title}>
```

Comment:
isn't the key attribute still needed though for react to do the iterating properly? is there no warning in the console about that now?

Reply:
It is needed, and in fact it is passed to the label, _but_ it is not a prop and therefore there's no need to make any reference to it in the label, React justs passes it down in the return statement of `AvailabilityZonesLabel` component. 

This is what I see in my `clusters-detail-node-pools`after merging this `fix-availability-zones-proptypes`branch:

<img width="440" alt="Captura de pantalla 2019-07-26 a las 8 24 02" src="https://user-images.githubusercontent.com/14203438/61930673-c0638b00-af7e-11e9-9632-45b4d91340ab.png">

Reply:
Nice! thanks :)